src/components/Builder/Breadcrumbs/Breadcrumbs.js

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import { Link } from "react-router-dom";
+
+const Breadcrumbs = ({ parent, parentText, rendered }) => {
+  return (
+    <p className="breadcrumb">
+      <Link className="breadcrumb__item" to="/">
+        Home
+      </Link>
+      /
+      <Link className="breadcrumb__item" to={ `/${parent}` } >
+        { parentText }
+      </Link>
+      /<span className="breadcrumb__item breadcrumb__item--selected">{rendered}</span>
+  </p>
```

Comment:
spacing is still off on the closing p tag
```suggestion
    <p className="breadcrumb">
      <Link className="breadcrumb__item" to="/">
        Home
      </Link>
      /
      <Link className="breadcrumb__item" to={ `/${parent}` } >
        { parentText }
      </Link>
      /<span className="breadcrumb__item breadcrumb__item--selected">{rendered}</span>
    </p>
```